docs/gallery/general/file.py

```diff
@@ -101,8 +101,8 @@
 # -------------------
 #
 # As with writing, reading is also carried out using the :py:class:`~pynwb.NWBHDF5IO` class.
-# To read the NWB file we just wrote, using construct another :py:class:`~pynwb.NWBHDF5IO` object,
-# and use the :py:func:`~pynwb.form.backends.io.FORMIO.read` method to retrieve an
+# To read the NWB file we just wrote, use another :py:class:`~pynwb.NWBHDF5IO` object,
+# and use the :py:func:`~pynwb.NWBHDF5IO.read` method to retrieve an
 # :py:class:`~pynwb.file.NWBFile` object.
 
 io = NWBHDF5IO('basic_example.nwb', 'r')
```

Comment:
It looks like this file open fails in the gallery tests.

Reply:
thanks